File: Eletronico.java
        // 2) Crie a classe Eletronico

public class Eletronico extends produto{
    private double voltagem;
    private double potencia;

    public double getVoltagem() {
        return voltagem;
    }

    public void setVoltagem(double voltagem) {
        this.voltagem = voltagem;
    }

    public double getPotencia() {
        return potencia;
    }

    public void setPotencia(double potencia) {
        this.potencia = potencia;
    }
}

File: Comex/Comex/modelo/Eletronico.cs
using System;
using System.Collections.Generic;
using System.Globalization;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Comex.modelo;

internal class Eletronico : Produto
{
    public Eletronico(string nome) : base(nome) { }
    
    public int Voltagem {  get; set; }
    public int Potencia { get; set; }

   
}

